server/sv_boss.lua

```diff
@@ -50,42 +50,43 @@ end)
-	local Employee = exports.qbx_core:GetPlayerByCitizenId(target)
+	local player = exports.qbx_core:GetPlayer(src)
+	local employee = exports.qbx_core:GetPlayerByCitizenId(target)
 
-	if not Player.PlayerData.job.isboss then ExploitBan(src, 'FireEmployee Exploiting') return end
+	if not player.PlayerData.job.isboss then ExploitBan(src, 'FireEmployee Exploiting') return end
 
-	if Employee then
-		if target ~= Player.PlayerData.citizenid then
-			if Employee.PlayerData.job.grade.level > Player.PlayerData.job.grade.level then exports.qbx_core:Notify(src, "You cannot fire this citizen!", "error") return end
-			if Employee.Functions.SetJob("unemployed", 0) then
-				TriggerEvent("qb-log:server:CreateLog", "bossmenu", "Job Fire", "red", Player.PlayerData.charinfo.firstname .. " " .. Player.PlayerData.charinfo.lastname .. ' successfully fired ' .. Employee.PlayerData.charinfo.firstname .. " " .. Employee.PlayerData.charinfo.lastname .. " (" .. Player.PlayerData.job.name .. ")", false)
+	if employee then
+		if target ~= player.PlayerData.citizenid then
+			if employee.PlayerData.job.grade.level > player.PlayerData.job.grade.level then exports.qbx_core:Notify(src, "You cannot fire this citizen!", "error") return end
+			if employee.Functions.SetJob("unemployed", 0) then
+				TriggerEvent("qb-log:server:CreateLog", "bossmenu", "Job Fire", "red", player.PlayerData.charinfo.firstname .. " " .. player.PlayerData.charinfo.lastname .. ' successfully fired ' .. employee.PlayerData.charinfo.firstname .. " " .. employee.PlayerData.charinfo.lastname .. " (" .. player.PlayerData.job.name .. ")", false)
 				exports.qbx_core:Notify(src, "Employee fired!", "success")
-				exports.qbx_core:Notify(Employee.PlayerData.source , "You have been fired! Good luck.", "error")
+				exports.qbx_core:Notify(employee.PlayerData.source , "You have been fired! Good luck.", "error")
 			else
 				exports.qbx_core:Notify(src, "Error..", "error")
 			end
 		else
 			exports.qbx_core:Notify(src, "You can\'t fire yourself", "error")
 		end
 	else
-		local player = MySQL.query.await('SELECT * FROM players WHERE citizenid = ? LIMIT 1', { target })
-		if player[1] then
-			Employee = player[1]
-			Employee.job = json.decode(Employee.job)
-			if Employee.job.grade.level > Player.PlayerData.job.grade.level then exports.qbx_core:Notify(src, "You cannot fire this citizen!", "error") return end
+		local offlineEmployee = MySQL.query.await('SELECT * FROM players WHERE citizenid = ? LIMIT 1', { target })
+		if offlineEmployee[1] then
+			employee = offlineEmployee[1]
+			employee.job = json.decode(employee.job)
+			employee.charinfo = json.decode(employee.charinfo)
+			if employee.job.grade.level > player.PlayerData.job.grade.level then exports.qbx_core:Notify(src, "You cannot fire this citizen!", "error") return end
 			local job = {}
 			job.name = "unemployed"
-			job.label = "Unemployed"
-			job.payment = jobs[job.name].grades['0'].payment or 500
+			job.label = "Civilian"
```

Comment:
Get the label the same way you got the payment; don't hardcode.

Reply:
Fixed